src/components/SelectWithDots/index.tsx

```diff
@@ -12,6 +12,31 @@ const DotsButton = styled(Button)`
   height: 35px;
 `
 
+type MenuAlignment = 'bottom-left' | 'bottom-right' | 'right-bottom' | 'right-top'
+
+const alignmentStyleOverrides: Record<MenuAlignment, React.CSSProperties> = {
```

Comment:
I think I want to move this back inline to where it's used since it's not obvious from up here what it's for